api/envoy/config/core/v3/protocol.proto

```diff
@@ -637,6 +637,14 @@ message Http3ProtocolOptions {
   // <https://datatracker.ietf.org/doc/draft-ietf-httpbis-h3-websockets/>`_
   // Note that HTTP/3 CONNECT is not yet an RFC.
   bool allow_extended_connect = 5 [(xds.annotations.v3.field_status).work_in_progress = true];
+
+  // [#not-implemented-hide:] Hiding until envoy has full metadata support.
+  // Still under implementation. DO NOT USE.
+  //
+  // Allows metadata. See [metadata
```

Comment:
nit: perhaps "Allow HTTP/3 METADATA frames".  `metadata` is kind of vague and could mean many different things.

Reply:
Done! (ditto)

Reply:
thanks Ryan!